data/rulesets.ts

```diff
@@ -388,6 +388,22 @@ export const Rulesets: {[k: string]: FormatData} = {
+			];
+			const species = this.dex.species.get(set.species || set.name);
+			if (!paldeaDex.includes(species.baseSpecies) && !paldeaDex.includes(species.name) &&
+				!this.ruleTable.has('+' + species.id)) {
```

Comment:
I know you're just copying from other stuff but I'm pretty sure this doesn't actually unban the pokemon